decision_desk/Dashboard/Dashboard Utilities/DemographicMaps.R

```diff
@@ -97,9 +97,9 @@ get_demographic_graph <- function(state_abbrev, demographic_type) {
 }
 
 state_abbrev <- "AL"
-demographic_type <- "Income"
+demographic_type <- "White"
 
-get_demographic_graph(state_abbrev, demographic_type)
+get_demographic_graph("~/GitHub/Election-Night", state_abbrev, demographic_type)
```

Comment:
don't love that the path is hard coded

Reply:
Why aren't we just requiring that everybody enters using the R project? I feel like that's clearly the most obvious solution to all of this